### ISSUE TITLE
[FRU] - Landing Page Frontend Template

### ISSUE THREAD

**Issue by danielfmiranda** [MozillaFoundation/foundation.mozilla.org#10266]
Once #10265 is complete, we are going to want to update the `LandingPage` model's template to follow the original donate sites design as close as possible.

![Screenshot 2023-04-18 at 11-44-18 Donate now Donate to Mozilla](https://user-images.githubusercontent.com/18314510/232874220-a2bea058-e2ef-43fd-b6d6-5fe710b50de4.png)

 ## Dev Tasks:
- [x] Create templates for the donate sites Navigation Bar and Footer. Style them to match the original donate site's design (with some adjustments. Mainly to make styling match our style guide. Changes have been approved by design team)
- [x] Update the landing page template to match the original donate site's design. (Similarly to above. Changes include some styling adjustments)

## Acceptance Criteria:
- [ ] As a user, when I visit the donate site's landing page, the design is the same as the original donate site (with some intentional styling tweaks)

**Comment by Gbaldassano** [comment]
@mmmavis to create separate ticket for Percy testing.

**Comment by mmmavis** [comment]
Hi @danielfmiranda , just wanted to check with you if I should be mocking what's on https://donate.mozilla.org/en-US/ now instead of the screencap included in ticket description.

**Comment by danielfmiranda** [comment]
Hi @mmmavis!

Thanks for checking and for the heads up! The screenshot above should have reflected the donate site with the updated fundraise up form, and I have updated the image in the description to accurately reflect what we are trying to recreate in this repo.

![Screenshot 2023-04-18 at 11-44-18 Donate now Donate to Mozilla](https://user-images.githubusercontent.com/18314510/232874135-083566b3-0d5c-473e-b884-4a90f349498e.png)

**Comment by mmmavis** [comment]
@danielfmiranda another q - do you know which FRU form I should use? Is it the same as what we have currently on https://donate.mozilla.org/ ?

**Comment by danielfmiranda** [comment]
Hi @mmmavis! 

Thanks for checking, it should be the same one that is being currently used on the donate site, its template on the donate repo can be found [here](https://github.com/MozillaFoundation/donate-wagtail/blob/main/donate/templates/fragments/donate_form_fundraise_up.html)

**Comment by mmmavis** [comment]
Didn't have enough time this sprint to properly wrap my assigned tickets for donate. The groundwork is done. I have the landing page built but there are still some remaining tasks

- work with design team to finalize design details
- simplify HTML a bit especially for the nav and footer. I mocked the DOM node structure with fo.mo existing code for now. If we are keeping the nav and footer on Donate pages simple, then there's no need for us to over complicate HTML code just so it matches the rest of the codebase.
- turn more SCSS code into Tailwind

**Comment by mmmavis** [comment]
Hey @danielfmiranda , can you advice how we QA this work on staging?

**Comment by danielfmiranda** [comment]
Hi @mmmavis!

This is quite confusing to QA on staging, as we do not yet have a subdomain set up for the donate site set up yet. For example, the mozfest pages have the site with a subdomain of "mozfest.mofostaging.net". However when I try to set up a site with donate.mofostaging.net I get a DNS error.

However, I have tested out the donate landing page by borrowing the mozfest subdomain and can confirm that everything is looking as expected! 

![image](https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/9a2e56ca-d88e-46ab-855f-cd0de77e0389)


This ticket is ready for prod :+1:

**Comment by mmmavis** [comment]
@danielfmiranda thanks for QAing this and sharing the workaround!
